assets/src/dashboard/app/views/my-stories/index.js

```diff
@@ -22,12 +22,43 @@ import { __ } from '@wordpress/i18n';
 /**
  * Internal dependencies
  */
-import { ViewHeader } from '../../../components';
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+import { useState } from 'react';
+import { ViewHeader, FloatingTab } from '../../../components';
+
+const FilterContainer = styled.div`
+  padding: 0 20px 20px;
+  border-bottom: 1px solid #eee;
+`;
+
+const storiesFilter = [
+  { label: 'All Stories', value: 'all-stories ' },
+  { label: 'Drafts', value: 'drafts' },
+  { label: 'Active Stories', value: 'active-stories ' },
+  { label: 'My Templates', value: 'my-templates ' },
+];
```

Comment:
I'm wondering - will we ever need to reference these elsewhere? If so, would it be better to store this in /constants.js?

Reply:
Perhaps! Not bad to move.